etc/kubernetes/manifests/base/apps/cos-fleetshard-sync/crds/managedconnectors.cos.bf2.org-v1.yml

```diff
@@ -19,7 +19,7 @@ spec:
     - jsonPath: .spec.connectorId
       name: CONNECTOR_ID
       type: string
-    - jsonPath: .spec.deployment.connectorTypeId
+    - jsonPath: .status.deployment.connectorTypeId
```

Comment:
I think this is not correct. It happens when the generator cannot figure the right JSON path.

Reply:
Do you have some pointer where/what to look at/for to figure out the issue?

Reply:
I've discussed this before with @lburgazzoli and that was our conclusion.